<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeVectorFormatConversionDialog</class>
 <widget class="QDialog" name="SeVectorFormatConversionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>线划图数据格式转换工具</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>543</width>
       <height>185</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_InputData">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>输入数据目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QPushButton" name="Button_Open">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_OdataPath">
           <property name="text">
            <string>odata格式的数据路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_InputDataPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_OutputData">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>输出数据目录</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="2">
          <widget class="QLineEdit" name="lineEdit_OutputDataPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_OutputPath">
           <property name="text">
            <string>shapefile格式的数据路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="Button_Save">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_OutputSRS">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>输出数据空间参考系</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_OriginSRS">
           <property name="text">
            <string>与输入数据保持一致</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_GeoSRS">
           <property name="text">
            <string>地理坐标系</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_ProjSRS">
           <property name="text">
            <string>投影坐标系</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_OffsetSetting">
        <property name="title">
         <string>转换偏移量设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_Scale">
           <property name="text">
            <string>比例尺分母</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="lineEdit_Scale">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_OffsetX">
           <property name="text">
            <string>X偏移量</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lineEdit_OffsetX">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_OffsetX_2">
           <property name="text">
            <string>Y偏移量</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="lineEdit_OffsetY">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_OffsetYUnit">
           <property name="text">
            <string>米</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_OffsetSetting_2">
        <property name="title">
         <string>坐标放大系数设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_read_from_sms_file">
           <property name="text">
            <string>从SMS文件读取</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_user_defined">
           <property name="text">
            <string>自定义</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_zome_Scale">
           <property name="text">
            <string>放大系数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEdit_zoom_scale">
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox_OffsetSetting_3">
        <property name="title">
         <string>获取图层方式设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_read_layer_info_from_odata_dir">
           <property name="text">
            <string>从odata分幅数据中实际存在的文件中读取图层信息</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton_read_layer_info_from_sms_file">
           <property name="text">
            <string>从odata分幅数据中的SMS文件读取图层信息</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QGroupBox" name="groupBox_log_level">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>日志设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_log_level_name">
           <property name="text">
            <string>日志等级</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox_level">
           <item>
            <property name="text">
             <string>消息</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>警告</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>关闭</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>调试</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>详细</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>一般错误</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>致命错误</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>688</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_log_level_name_2">
           <property name="text">
            <string>汇总日志保存路径</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLineEdit" name="lineEdit_summary_log_saving_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="Button_summary_log_saving_path">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>确认转化</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="Button_OK">
           <property name="text">
            <string>开始转换</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="Button_Cancel">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
